File: mcs/tools/corcompare/transform.js
var xml = WScript.CreateObject ("MSXML2.DOMDocument");
var xsl = WScript.CreateObject ("MSXML2.DOMDocument");

xml.async = false;
xsl.async = false;

xml.load (WScript.Arguments (0));
xsl.load (WScript.Arguments (1));

WScript.Echo (xml.transformNode (xsl));

File: mcs/tools/corcompare/transform.cs
using System;
using System.Xml;
using System.Xml.Xsl;

namespace Transform
{
	public class Transform
	{
		public static void Main (string [] rgstrArgs)
		{
			XmlReader xml = XmlReader.Create (rgstrArgs [0]);

			XslCompiledTransform xsl = new XslCompiledTransform ();
			xsl.Load (rgstrArgs [1]);

			xsl.Transform (xml, null, Console.Out);
		}
	}
}
